client/store/modules/repositories/actions.js

```diff
@@ -45,8 +45,11 @@ const pin = ({ commit, getters }, { id, pin }) => {
   });
 };
 
-const fetchTags = ({ commit }) => {
-  return tagApi.fetch().then(tags => commit('fetchTags', tags));
+const fetchTags = ({ commit }, payload) => {
```

Comment:
No need to modify this one since store will only have assigned ones (non assigned are only being used / and fetched within modal).